### PR TITLE
Add base survey questions

### PR DESCRIPTION
**Pull request by megpaik**

### CODE REVIEW

File: common/schema.js [hunk 1]
```diff
@@ -33,73 +33,394 @@ const submissionStatusArray = Object.values(submissionStatus);
...
+      'Yes',
+      'No',
+      'Prefer not to say',
+    ],
+    required: true,
+    type: [String],
+  },
+];
+
+// TODO: make this a function that can take custom params
```
Comment (megpaik) [inline common/schema.js:146]: I wasn't sure what this meant, lmk if I should be doing something else with these q's!

**Review by gfting — CHANGES_REQUESTED** [review]
Note that `[String]` means it's anticipating an array of strings rather than a single `String` in the case of a radio component. Also, I'm not convinced that making things "q1" and "q2" is correct, as it'll make reordering inflexible in the future.

**Comment by megpaik** [comment]
updated Radio types. Can you clarify your concern with the id naming? My understanding was that the id would simply serve as a unique identifier - how does this affect ordering?

**Comment by gfting** [comment]
> updated Radio types. Can you clarify your concern with the id naming? My understanding was that the id would simply serve as a unique identifier - how does this affect ordering?

Thanks. What I was thinking is that it's inflexible for the future–consider that new questions get added, or there's a new numbering. In that scenario, the numbers for `q${i}` would no longer be accurate.

**Review by gfting — APPROVED** [review]

**Pull request merged**